src/styles/blocks/star.scss

```diff
@@ -0,0 +1,22 @@
+.stars {
+  display: flex;
+  width: 96px;
+  margin-left: 18px;
+
+  &__star {
+    background-image: url(../images/star.svg);
+    background-repeat: no-repeat;
+    width: 16px;
+    height: 16px;
+    margin-right: 4px;
+  }
+
+  &__star:nth-child(-n + 4) {
+    margin-right: 0;
```

Comment:
You have hard-coded the number of active stars to 4. This is not flexible. The number of active stars should be determined dynamically, based on the rating of the product.